shard.go

```diff
@@ -253,6 +254,15 @@ func (s *cacheShard) onEvict(oldestEntry []byte, currentTimestamp uint64, evict
 }
 
 func (s *cacheShard) cleanUp(currentTimestamp uint64) {
+	defer func() {
+		// panic recover
```

Comment:
Why do we need this?

Reply:
In https://github.com/allegro/bigcache/issues/226 and https://github.com/allegro/bigcache/issues/148, It panic in `cleanUp`.

If CleanWindows is not zero, bigcache starts a goroutine, if cleanUp panic, the whole program will be exited. https://github.com/allegro/bigcache/blob/bbbffd30d25b538b4c03be6649b1dffbacca9596/bigcache.go#L85-L99

This is a temporary solution to prevent crash of whole program, actually, find out the reason why panic is important.